worker/src/lib.rs

```diff
@@ -52,6 +53,7 @@ mod cache;
 mod cf;
 mod context;
 mod cors;
+mod d1;
```

Comment:
Needs to be gated behind a feature flag.

Reply:
Good idea, will implement the feature flag.